sys/net/gnrc/routing/rpl/p2p/gnrc_rpl_p2p.c

```diff
@@ -0,0 +1,390 @@
+/*
+ * Copyright (C) 2015 Cenk Gündoğan <cnkgndgn@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ *
+ * @author  Cenk Gündoğan <cnkgndgn@gmail.com>
+ */
+
+#include "net/icmpv6.h"
+#include "net/gnrc/ipv6.h"
+#include "net/gnrc/icmpv6.h"
+#include "net/gnrc.h"
+#include "net/gnrc/rpl/structs.h"
+#include "net/gnrc/rpl/dodag.h"
+#include "net/gnrc/rpl/p2p.h"
+#include "net/gnrc/rpl/p2p_structs.h"
+#include "net/gnrc/rpl/p2p_dodag.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+#if ENABLE_DEBUG && defined(MODULE_IPV6_ADDR)
```

Comment:
I just stuck on this one, `&& defined(MODULE_IPV6_ADDR)` is always `true` for this implementation, since you use `ipv6_addr_t`